bigml/local_model.py

```diff
@@ -0,0 +1,234 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=super-init-not-called
+#
+# Copyright 2023 BigML
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+"""A local Predictive model class abstracting all kind of models
+
+This module abstracts any BigML model to make predictions locally or
+embedded into your application without needing to send requests to
+BigML.io.
+
+This module cannot only save you a few credits, but also enormously
```

Comment:
do we still have credits? I think it is something that we used in the past but we don't use any more. Isn't it?

Reply:
That's exactly it. We deprecated credits a long time ago.